modules/waypoint_tracking.py

```diff
@@ -0,0 +1,33 @@
+"""
+Module for obtaining information about the drone,
+including the current waypoint sequence, location, and destination.
+"""
+import dronekit
+
+
+def get_current_location(drone) -> "tuple[float,float]":
```

Comment:
Return `"tuple[bool, tuple[float, float]"` where `False, None` is failure and `True, [data]` is success.